app/views/unattended/provisioning_templates/snippet/puppet_setup.erb

```diff
@@ -55,9 +85,8 @@ else
   yum -t -y install <%= linux_package %>
 fi
 <% elsif os_family == 'Suse' -%>
-<% if host_param_true?('enable-puppetlabs-repo') || host_param_true?('enable-official-puppet8-repo') || host_param_true?('enable-official-puppet7-repo') || host_param_true?('enable-puppetlabs-puppet6-repo') || host_param_true?('enable-puppetlabs-puppet5-repo') -%>
-rpmkeys --import https://yum.puppet.com/RPM-GPG-KEY-puppetlabs
-rpmkeys --import https://yum.puppet.com/RPM-GPG-KEY-puppet
+<% if host_param_true?('openvox-version') || host_param_true?('enable-openvox-repo') -%>
+rpmkeys --import https://yum.voxpupuli.org/GPG-KEY-openvox.pub
```

Comment:
I don't know SuSE that well, but I think that import was needed because some oddities on the dual key use. Right now there are no SuSE instructions on https://voxpupuli.org/openvox/install/ at all so I'd be tempted to leave it out.